src/app/models/UnidadeReferencia.js

```diff
@@ -0,0 +1,20 @@
+import Sequelize, { Model } from 'sequelize';
+
+
+class UnidadeReferencia extends Model {
+    static init(sequelize) {
+        super.init(
+            {
+                descricao: Sequelize.STRING,
+            },
+            {
+                sequelize,
+            }
+        );
+
+        return this;
+
+    }
+
```

Comment:
Unidade de referência hasMany cadastros -> para facilitar busca

Reply:
Dinho, fiquei na dúvida de como fazer esse. Ficaria assim? this.hasMany(models.Cadastro, { foreignKey: 'id_usuario', as: 'usuario' }); Eu acho que não pq esse id_usuario estaria relacionado a usuario e não ao cadastro. Mas não sei a qual foreignKey relacionar, seria só a 'id'?  Fiquei na dúvida

Reply:
a unidade de referência tem muitos cadastros e os cadastros se relacionam com a unidade de referência pela chave id_unidade se não me engano.